multimedia.py

```diff
@@ -35,15 +47,15 @@
 
 
 
-@app.route('/')
+
 def home():
     return render_template('index.html')
 
 @app.route('/new_User')
 def newPage():
     return render_template('introPage.html')
 
-@app.route('/returing_User', methods=['GET', 'POST'])
+@app.route('/', methods=['GET', 'POST'])
```

Comment:
should this stay as it is for returning users. Also something minor is the spelling.

Reply:
no that becomes the new home route